### ISSUE TITLE
RSS feed

### ISSUE THREAD

**Issue by stp-ip** [okkur/syna#262]
**Is this a BUG REPORT or FEATURE REQUEST?**:
feature

**What happened**:
No global nor per section feed is available.

**What you expected to happen**:
Per section and one global feed available via RSS.

**Comment by mpourismaiel** [comment]
Removing RSS from `disableKinds` and adding it to `outputs.home` would resolve the issue and generate a decent rss feed, accessible by `/index.xml` and `<section/index.xml`.

**Comment by stp-ip** [comment]
That would mean a global rss feed of all content pages right?

**Comment by mpourismaiel** [comment]
Yeah, I'll create a PR and we can see if it requires anything else.
